dali/pipeline/data/types.h

```diff
@@ -325,6 +325,7 @@ DALI_REGISTER_TYPE(int16, DALI_INT16);
 DALI_REGISTER_TYPE(int32, DALI_INT32);
 DALI_REGISTER_TYPE(int64, DALI_INT64);
 DALI_REGISTER_TYPE(float16, DALI_FLOAT16);
+DALI_REGISTER_TYPE(float16_cpu, DALI_FLOAT16);
```

Comment:
Please no.
We have bijective mapping defined here, which you break by giving the enum for two distinct types,
which will break at this point overwriting previous entry in this map: https://github.com/NVIDIA/DALI/blob/b3f75f3c2e4fadc9a062f4edfb8e350dc7acbc06/dali/pipeline/data/types.h#L239

Reply:
Same issue as in #209

Reply:
Fixed

Reply:
I don't feel quite entitled to review this, since this directly follows my suggestion.